sparkle/src/components/Page.tsx

```diff
@@ -134,6 +135,59 @@ const gapSizes = {
   none: "",
 };
 
+Page.Layout = function ({
+  children,
+  direction = "vertical",
+  sizing,
+  align = "left",
+  gap = "lg",
+}: PageLayoutProps) {
+  switch (direction) {
+    case "horizontal":
+      return (
+        <Page.Horizontal
+          children={children}
+          sizing={sizing}
+          align={align}
+          gap={gap}
+        />
+      );
+    case "vertical":
+      return (
+        <Page.Vertical
+          children={children}
+          sizing={sizing}
+          align={align}
+          gap={gap}
+        />
+      );
+    case "fluid":
+      return (
+        <Page.Fluid
+          children={children}
+          sizing={sizing}
+          align={align}
+          gap={gap}
+        />
+      );
+    default:
+      return (
+        <Page.Vertical
+          children={children}
+          sizing={sizing}
+          align={align}
+          gap={gap}
+        />
```

Comment:
As direction is typed, we should never be there. We can return null; throw an error, or keep as is but remove the `case "vertical":` that is duplicated?

Reply:
Ok! Thought having a default was mandatory. Changing.

Reply:
@Duncid having a default is mandatory I suggested to return null instead, or keep the default and remove case vertical !